app/models/dashboard/postgres_response.rb

```diff
@@ -13,6 +13,7 @@ def documents
 
       def work_versions
         WorkVersion.where(uuid: work_version_uuids)
+          .order(updated_at: :desc)
```

Comment:
I think this works for now, but it will break when we start interlacing Collections into the mix. In that case we'll need to do the ordering in the Solr query, and then maintain the order of the results when we map them to ActiveRecords... but we will _already_ need to do that to add collections into the list, so maybe that will inherently sort itself out (pun intended)